Documentation/guides/simulator.rst

```diff
@@ -21,50 +23,100 @@ Compiling
    There are a lot of simulator configurations available that set you up to test various
    operating system features.
 
+   Here we'll use the ``sim:nsh`` basic NuttX Shell configuration.
+
```

Comment:
Shouldn't there be a command line example below?

Reply:
@v01d Yeah this was a copy paste error, there was some extra stuff there that shouldn't have been there.